### PR TITLE
[Term Entry] C++ Math Functions: sinh() #980

### PR DESCRIPTION
**Pull request by Hariprasad19036**
### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cpp/concepts/math-functions/terms/sinh/sinh.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+Returns the hyperbolic sine of an argument given in radians.
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/sinh/sinh.md:14]:
```suggestion
The **`sinh()`** function returns the hyperbolic sine of an argument given in radians.
```

File: content/cpp/concepts/math-functions/terms/sinh/sinh.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Syntax
+
+```cpp
+std::sinh(n)
+```
+## Example 1
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/sinh/sinh.md:21]:
```suggestion

## Example
```

File: content/cpp/concepts/math-functions/terms/sinh/sinh.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Use `sinh()` to return the hyperbolic cosine of `0.0` radians:
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/sinh/sinh.md:22]:
```suggestion

Use `sinh()` to return the hyperbolic sine of `0.0` radians:

```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Hariprasad19036, I did a first review and left a couple of comments.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Hariprasad19036 looks like we're ready for a second review.

**Review by SSwiniarski — APPROVED** [review]

File: content/cpp/concepts/math-functions/terms/sinh/sinh.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`sinh()`** function returns the hyperbolic sine of an argument given in radians.
+
+## Syntax
+
+```cpp
+std::sinh(n)
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/sinh/sinh.md:19]:
```suggestion
std::sinh(angle)
```

File: content/cpp/concepts/math-functions/terms/sinh/sinh.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+```
+
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/sinh/sinh.md:21]:
```suggestion

The `angle` parameter should be expressed in radians. To convert from degrees to radians:

```

And then I would add one more \```pseudo code snippet that details how to do this:

```pseudo
radians = degrees * PI / 180
```

File: content/cpp/concepts/math-functions/terms/sinh/sinh.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+## Example
+
+Use `sinh()` to return the hyperbolic sine of `0.0` radians:
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/sinh/sinh.md:24]:
```suggestion
The following example uses `sinh()` to return the hyperbolic sine of `0.0` in radians:
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@Hariprasad19036 Finished with my review and left you some comments/suggestions. 😄

File: content/cpp/concepts/math-functions/terms/sinh/sinh.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+The **`sinh()`** function returns the hyperbolic sine of an argument given in radians.
+
+## Syntax
+
+```cpp
+sinh(angle)
+```
+
+The `angle` parameter should be expressed in radians. To convert from degrees to radians:
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/sinh/sinh.md:22]: We seem to be missing the other side of the colon `:`?

**Review by Dusch4593 — CHANGES_REQUESTED** [review]

File: content/cpp/concepts/math-functions/terms/sinh/sinh.md (hunk 7 above)
Comment (Hariprasad19036) [reply content/cpp/concepts/math-functions/terms/sinh/sinh.md:22]: Done, Kindly check

File: content/cpp/concepts/math-functions/terms/sinh/sinh.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`sinh()`** function returns the hyperbolic sine of an argument given in radians.
+
+## Syntax
+
+```cpp
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/sinh/sinh.md:18]:
```suggestion
```pseudo
```

File: content/cpp/concepts/math-functions/terms/sinh/sinh.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+sinh(angle)
+```
+
+The `angle` parameter should be expressed in radians. To convert from degrees to radians:
+
+```
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/sinh/sinh.md:24]:
```suggestion
```pseudo
```

File: content/cpp/concepts/math-functions/terms/sinh/sinh.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+## Example
+
+The following example uses `sinh()` to return the hyperbolic sine of `0.0` in radians:
+
+```cpp
+#include <iostream>
+#include <cmath>
+int main() {
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/sinh/sinh.md:35]:
```suggestion
#include <cmath>

int main() {
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@Hariprasad19036 Just a few last small suggestions from me. 😄

**Review by Dusch4593 — APPROVED** [review]
@Hariprasad19036 LGTM! 😄

**Pull request merged**